doc/nrf/ug_multiprotocol_support.rst

```diff
@@ -0,0 +1,52 @@
+.. _ug_multiprotocol_support:
+
+Multiprotocol support for Thread and Zigbee
+###########################################
+
+The :ref:`ug_thread` and :ref:`ug_zigbee` protocols support running another protocol in parallel.
+
+This is achieved with the :ref:`nrfxlib:softdevice_controller`, an implementation of the :ref:`ug_ble_controller`.
+The SoftDevice Controller in turn relies on the functionalities of ref:`Multi-Protocol Service Layer (MPSL) <nrfxlib:mpsl>` library, which provides services for multiprotocol applications and allows the 802.15.4 radio driver to negotiate for transmission timeslots with the controller.
```

Comment:
Actually, it is achieved by the implementation of MPSL. Any radio protocol using MPSL to schedule radio access can be used in parallel with thread and zigbee. Sd controller is just one example of such protocol. I mean, sd controller is not needed for parallel radio operations. Only MPSL is needed. SD controller is just an example of other radio protocol that can run in parallel with thread or zigbee

Reply:
I guess that SD controller is the only one which uses MPSL in NCS. We can not run for instance Zephyr BLE controller together with thread or zigbee. From what I understand in theory someone can provide it's own protocol stack which uses MPSL and based on that can run thread/zigbee concurrently.

Reply:
Rephrased.

Reply:
Yes, the only protocols provided by nordic in ncs that support f dynamic multiprotocol are 802.15.4 driver and sd controller. However, a customer could implement a proprietary radio protocol using Timeslot API from MPSL and use dynamic multiprotocol with 802.15.4, without BLE controller.
Because of that I suggest to remove text indicating that dynamic multiprotocol is implemented in SD Ctlr.